app/src/org/commcare/activities/connect/ConnectIdActivity.java

```diff
@@ -0,0 +1,166 @@
+package org.commcare.activities.connect;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+
+import org.commcare.android.database.connect.models.ConnectUserRecord;
+import org.commcare.connect.ConnectConstants;
+import org.commcare.connect.ConnectDatabaseHelper;
+import org.commcare.connect.ConnectManager;
+import org.commcare.dalvik.R;
+import org.commcare.fragments.connectId.ConnectIdBiometricConfigFragment;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.fragment.app.Fragment;
+import androidx.navigation.NavController;
+import androidx.navigation.NavDirections;
+import androidx.navigation.fragment.NavHostFragment;
+
+
+public class ConnectIdActivity extends AppCompatActivity {
+    static NavController controller;
+    static NavDirections navDirections = null;
+    public static boolean forgotPassword = false;
+    public static boolean forgotPin = false;
+    public static String recoverPhone;
+    public static String recoverSecret;
+    public static String recoveryAltPhone;
+
+    @Override
+    protected void onActivityResult(int requestCode, int resultCode, Intent data) {
+        super.onActivityResult(requestCode,resultCode,data);
+        if(requestCode==ConnectConstants.CONNECT_UNLOCK_PIN) {
+            getCurrentFragment().onActivityResult(requestCode, resultCode, data);
+        }
+        if(requestCode==ConnectManager.CONNECTID_REQUEST_CODE){
```

Comment:
"else if"

Also seems strange that this constant is defined in ConnectManager instead of ConnectConstants like the one above.